frontend/src/app/profile/page.js

```diff
@@ -0,0 +1,103 @@
+// import { useState } from 'react';
+import Sidebar from '../../components/sidebar';
+import Image from 'next/image';
+import './profile.css';
+
+export default function Profile() {
+//   const [profilePic, setProfilePic] = useState('/profile-pic.jpg'); // Default profile picture
+//   const [view, setView] = useState('posts'); // State to manage selected view
+
+//   const handleImageChange = (event) => {
+//     const file = event.target.files[0];
+//     if (file) {
+//       const reader = new FileReader();
+//       reader.onloadend = () => {
+//         setProfilePic(reader.result);
+//       };
+//       reader.readAsDataURL(file); // Convert image to base64 URL
+//     }
+//   };
+
+  return (
+    <div style={{ display: 'flex', backgroundColor:'#37474f'}}>
+      <Sidebar />
+      <div className='content'>
+        <header className='header'>
+          <h1>SearchBar?</h1>
+        </header>
+        <main className='body'>
+          <div className='profileInfo'>
+            <label htmlFor='profilePicInput' className='profileImageLabel'>
+              <img
```

Comment:
Similar to the Sidebar component, img is used instead of Image from next/image.

Reply:
I believe in this file I didn't use next/image. I switched to just using img because the other method required me to alter next.config